src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -87,6 +87,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->booleanNode('enable_swagger')->defaultValue(true)->info('Enable the Swagger documentation and export.')->end()
                 ->booleanNode('enable_swagger_ui')->defaultValue(class_exists(TwigBundle::class))->info('Enable Swagger ui.')->end()
+                ->booleanNode('enable_entrypoint')->defaultTrue()->info('Enable/disable the entry-point')->end()
```

Comment:
`Enable the entrypoint` for the shake of consistency

Reply:
I would also create a separate flag for entrypoint and docs, (you may want the docs, but not the entrypoint for instance).